src/views/PermissionSend.vue

```diff
@@ -210,6 +212,21 @@ export default {
         }
       }
     },
+    async estimateGas() {
```

Comment:
Can you ensure that this is functioning correctly for other injections?
This permission screen i believe is shared among others so i'm wondering about something ETH specific being introduced here.

For example try a bitcoin send

some simple docs on bitcoin injection :

https://liquality.io/documentation/web_extension/bitcoin/getting_started/

Reply:
Tested and verified, works well for BTC